accelerator/real_accelerator.py

```diff
@@ -65,34 +65,53 @@ def get_accelerator():
             except ImportError as e:
                 raise ValueError(
                     f'CPU_Accelerator requires intel_extension_for_pytorch, which is not installed on this system.')
+        elif accelerator_name == 'npu':
+            try:
+                import torch_npu  # noqa: F401
+            except ImportError as e:
+                raise ValueError(
+                    f'NPU_Accelerator requires torch_npu, which is not installed on this system.')
+            pass
         elif accelerator_name == 'cuda':
             pass
         else:
             raise ValueError(
-                f'DS_ACCELERATOR must be one of "cuda", "cpu", or "xpu".  Value "{accelerator_name}" is not supported')
+                f'DS_ACCELERATOR must be one of "cuda", "cpu", "xpu", or "npu".  Value "{accelerator_name}" is not supported')
```

Comment:
Could we introduce a global list of accelerator names and use that here?
I mean something like the following

```python
DS_ACCELERATOR_LIST = ['cuda', 'cpu', ...]

...
raiseValueError(f'DS_ACCELERATOR environment variable must be one of {DS_ACCELERATOR_LIST} ...'
```

Reply:
Sure. Will change to use global list of accelerators next push.